app/controllers/users_controller.rb

```diff
@@ -33,4 +46,9 @@ def require_login
       redirect_to login_url
     end
   end
+
+  def correct_user
+    @user = User.find(params[:id])
```

Comment:
You are repeating `@user = User.find(params[:id])` you can do:
```ruby
private
...
def user
   @user = User.find(params[:id])
end
```
Or just assign a different method name like `initialize_user` and use the callback `before_action :initialize_user`
```ruby
before_action :initialize_user
...
private
...
def initialize_user
   User.find(params[:id])
end
```
This will become better if you need to add more methods.

Reply:
Thanks, I will add the 2nd approach